src/services/api/pets/contracts.ts

```diff
@@ -0,0 +1,31 @@
+export enum Gender {
+  Female = 'female',
+  Male = 'male',
+  Undefined = 'undefined',
+}
+
+export enum PetType {
+  Cat = 'cat',
+  Dog = 'dog',
+  Horse = 'horse',
+  Undefined = 'undefined',
+}
+
+export interface CreatePetDto {
```

Comment:
`DTO` is an acronym, therefore, we should use all uppercase within names 
```suggestion
export interface CreatePetDTO {
```

Reply:
Who told you that? In books i have code that have "Dto" in names. Do you know any rules for namig acronyms in your way?

Reply:
Ok, I've started some digging, and it looks that I've remembered only half of the rule. By referring to a similar discussion [here](https://stackoverflow.com/questions/15526107/acronyms-in-camelcase) I agree with you that `Dto` is an acceptable form. Only two-letter acronyms or abbreviations should use the `UPPERCASE` form.

I guess it's my personal taste that tells me that I'm butchering the acronym by using lowercase letters.